test/apiv2/47-subnet-pools.at

```diff
@@ -0,0 +1,36 @@
+# -*- sh -*-
+#
+# subnet pools conversion tests
+#
+
+# Create a temporary containers.conf with custom DefaultSubnetPools
+tempdir=$(mktemp -d)
+cat > $tempdir/containers.conf <<EOF
+[network]
+default_subnet_pools = [
+  { "base" = "10.89.0.0/16", "size" = 24 },
+  { "base" = "10.90.0.0/15", "size" = 26 }
+]
+EOF
+
+# Start podman with custom config
+CONTAINERS_CONF=$tempdir/containers.conf
+
+# Test the Docker-compatible API endpoint with our custom configuration
+t GET system/info 200
+
+# Confirm DefaultAddressPools exists
+t GET system/info 200 .DefaultAddressPools
+
+# Verify the number of pools (should be 2)
+t GET system/info 200 '.DefaultAddressPools | length=2'
+
+# Verify the pools
+t GET system/info 200 '.DefaultAddressPools[0].Base="10.89.0.0/16"'
+t GET system/info 200 '.DefaultAddressPools[0].Size=24'
+
+t GET system/info 200 '.DefaultAddressPools[1].Base="10.90.0.0/15"'
+t GET system/info 200 '.DefaultAddressPools[1].Size=26'
```

Comment:
This is inefficient as you do a ton of info requests, you can chain the test strings

```
t GET system/info 200 \
  '.DefaultAddressPools | length=2' \
  '.DefaultAddressPools[0].Base="10.89.0.0/16"' \
  '.DefaultAddressPools[0].Size=24' \
  '.DefaultAddressPools[1].Base="10.90.0.0/15"' \
  '.DefaultAddressPools[1].Size=26'
```

Reply:
addressed. Thanks